deltachat-jsonrpc/src/api.rs

```diff
@@ -1205,7 +1205,15 @@ impl CommandApi {
     async fn delete_messages(&self, account_id: u32, message_ids: Vec<u32>) -> Result<()> {
```

Comment:
Maybe metion that this doesn't delete messages for all? "See `delete_messages_for_all`".